packages/vue/src/components/organisms/SfHeader/SfHeader.vue

```diff
@@ -1,115 +1,135 @@
-              class="sf-header__search"
-              @input="$emit('change:search', $event)"
-              @enter="$emit('enter:search', $event)"
+            <SfIcon
+              :icon="accountIcon"
+              size="1.25rem"
+              :class="{
+                'sf-header__icon is-active': activeIcon === 'account',
+              }"
             />
-          </slot>
-          <!--@slot Use this slot to replace default header icons with custom content-->
-          <slot
-            name="header-icons"
-            v-bind="{
-              activeIcon,
-              cartHasProducts,
-              cartItemsQty,
-              cartIcon,
-              wishlistIcon,
-              accountIcon,
-            }"
+          </SfButton>
+          <SfButton
+            v-if="wishlistIcon"
+            class="sf-button--pure sf-header__action"
+            data-testid="wishlistIcon"
+            @click="$emit('click:wishlist')"
           >
-            <div class="sf-header__icons">
-              <SfButton
-                v-if="accountIcon"
-                class="sf-button--pure sf-header__action"
-                data-testid="accountIcon"
-                @click="$emit('click:account')"
-              >
-                <SfIcon
-                  :icon="accountIcon"
-                  size="1.25rem"
-                  :class="{
-                    'sf-header__icon is-active': activeIcon === 'account',
-                  }"
-                />
-              </SfButton>
-              <SfButton
-                v-if="wishlistIcon"
-                class="sf-button--pure sf-header__action"
-                data-testid="wishlistIcon"
-                @click="$emit('click:wishlist')"
-              >
-                <SfIcon
-                  class="sf-header__icon"
-                  :icon="wishlistIcon"
-                  :has-badge="wishlistHasProducts"
-                  :badge-label="wishlistItemsQty"
-                  size="1.25rem"
-                  :class="{
-                    'sf-header__icon is-active': activeIcon === 'wishlist',
-                  }"
-                />
-              </SfButton>
-              <SfButton
-                v-if="cartIcon"
-                class="sf-button--pure sf-header__action"
-                data-testid="cartIcon"
-                @click="$emit('click:cart')"
-              >
-                <SfIcon
-                  class="sf-header__icon"
-                  :icon="cartIcon"
-                  :has-badge="cartHasProducts"
-                  :badge-label="cartItemsQty"
-                  size="1.25rem"
-                  :class="{
-                    'sf-header__icon is-active': activeIcon === 'cart',
-                  }"
-                />
-              </SfButton>
-            </div>
-          </slot>
+            <SfIcon
+              class="sf-header__icon"
+              :icon="wishlistIcon"
+              size="1.25rem"
+              :class="{
```

Comment:
Same here. You can also create a method that will accept a menu item name, i.e. `wishlist`